blueprints/data-solutions/dataproc-lifecycle-via-composer/terraform/variables.tf

```diff
@@ -0,0 +1,73 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+variable "project_id" {
+  description = "Project ID where all the resources will be created, including; Cloud Composer Environment, dataproc cluster, policies, buckets, etc."
+  default     = "pso-amex-data-platform"
```

Comment:
we never use a default for the project id, also be way about actual customer names...

Reply:
Thank you for reviewing this!
default project id removed.